content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
 {{< code/copyable >}}
 {
   "require": {
-    "php": ">=7.1"
+    "php": ">=8.0"
```

Comment:
This bit says 8.0 but the explainer text above says v7.1. I can't add a "suggestion" that fixes bc it's not in the PR diff.